deploy/helm/galaxy/templates/deployment.yaml

```diff
@@ -37,9 +37,30 @@ spec:
           {{- end }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if and .Values.externalSecrets.enabled .Values.externalSecrets.allAsEnv}}
+          env:
+            {{- include "galaxy-helm.base.envs" . | nindent 12 }}
```

Comment:
Nitpick: couldn't this be kept outside of the if/else block?

It seems to always be set, regardless of what value that condition evaluates to

Reply:
These should always be set that's why